libnetwork/Host.cpp

```diff
@@ -195,7 +198,23 @@ std::function<bool(bool, boost::asio::ssl::verify_context&)> Host::newVerifyCall
                                << LOG_KV("nodeID", nodeID.substr(0, 4));
                 return false;
             }
+            if (!preverified)

```

Comment:
this is unnecessary, what about delete it?

Reply:
This condition make the callback return false early if the certificate is invalid to in case of parse the common name for the certificate.

Reply:
if the certificate is invalid, then `cert  == NULL`, this will not run.
if certificate is wrong, then do below will not lead to error.
if this `if (!preverified)` is necessary, it should in the front of this lambda.

Reply:
done